docs/src/examples/components/Flex/Types/FlexColumnsExample.shorthand.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import { Flex, Segment } from '@stardust-ui/react'
+
+const FlexExampleShorthand = () => (
+  <>
+    <Flex gap="gap.small" padding="padding.medium">
+      <Flex.Item size="size.half">
+        <Segment content="1/2" />
+      </Flex.Item>
+
+      <Flex.Item size="size.half">
+        <Segment content="1/2" />
+      </Flex.Item>
+    </Flex>
+
+    <Flex gap="gap.small" padding="padding.medium">
+      <Flex.Item size="size.quarter">
+        <Segment content="1/4" />
+      </Flex.Item>
+
+      <Flex.Item size="size.half">
+        <Segment content="1/2" />
+      </Flex.Item>
+
+      <Flex.Item size="size.quarter">
+        <Segment content="1/4" />
+      </Flex.Item>
+    </Flex>
+
+    <Flex gap="gap.small" padding="padding.medium" style={{ minHeight: 200 }}>
```

Comment:
We should not confuse users with the `style` prop on Stardust elements, however I don't have a proposal there

Reply:
agree - let me try to address it